core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaCustomPayloadTranslator.java

```diff
@@ -138,6 +141,14 @@ public void translate(GeyserSession session, ClientboundCustomPayloadPacket pack
 
                 session.sendUpstreamPacket(toSend);
             });
+        } else if (channel.equals(PluginMessageChannels.COMMANDS)) {
+            session.ensureInEventLoop(() -> {
+                byte[] data = packet.getData();
+                String dataString = new String(data, StandardCharsets.UTF_8);
+
+                Map<String, String> parsedCommandDescriptions = new Gson().fromJson(dataString, new TypeToken<Map<String, String>>(){}.getType());
```

Comment:
This doesn't need to be JSON. See: https://github.com/Camotoy/Geyser/commit/cf8eb0d1a016533e94ea7f55bb57d6b91aac1cdb#diff-102f70618eda60c10b2a4991aaa3ce1551f602a9f970442fa9dfffceeae6c18bR59